src/main/java/org/fundacionjala/coding/ariel/EAN.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.ariel;
+
+/**
+ * Ean checksum.
+ * By Ariel Gonzales.
+ */
+public class EAN {
+
+    public static final int LASTDIGIT = 12;
+    public static final int PAR = 3;
+    public static final int CHECKSUM1 = 10;
+
+    /**
+     * @param number A string number.
+     * @return if the checksum is validate.
+     */
+    public boolean validate(String number) {
+        String[] numeros = number.split("");
+        int suma = 0;
+        for (int i = 0; i < numeros.length - 1; i++) {
+            int num = Integer.parseInt(numeros[i]);
+            suma += (i % 2 == 0) ? num : num * PAR;
+        }
+        int checksum = CHECKSUM1 - (suma % CHECKSUM1);
```

Comment:
the calculation for checksum is wrong 
Please double check.
and also you will need to add one unit test.